Sources/GRInputField/UIKit/InputFieldView.swift

```diff
@@ -563,7 +564,12 @@ public extension InputFieldView {
 
     var isSelected: Bool { textField.isSelected }
 
-    func setup(with model: Model) {
+    func setup(with model: Model, customAppearance: InputFieldAppearance? = nil) {
+        if let customAppearance {
+            standardAppearance = customAppearance
+            setupAppearance()
```

Comment:
I'm adding a suggestion - separating the appearance from behaviour. This is so that existing implementations in UIKit can modify the appearance in their own way (for example: one can modify the appearance by subclassing, accessing the properties, assigning a new appearance proxy or even by modifying the UIKit hierarchy directly). We've got all kinds of mess in our projects, and I hope this should make it more compatible.

Also it supports the "separation of responsibilities" principle, since setting the appearance will be its own function, decoupled from the "model"/traits of the input field.

```suggestion
    func setupAppearance(_ appearance: InputFieldAppearance) {
        standardAppearance = customAppearance
        setupAppearance()
    }

    func setup(with model: Model) {
```